stdlib/public/runtime/HeapObject.cpp

```diff
@@ -28,7 +28,9 @@
 #include <cstring>
 #include <cstdio>
 #include <cstdlib>
+#if !defined(_MSC_VER)
 #include <unistd.h>
```

Comment:
It seems like we are not using this include for anything, could you try removing it?